### ISSUE TITLE
Can't bring the text in Firenvim overlay back to Text Area!

### ISSUE THREAD

**Issue by kuochang** [glacambre/firenvim#408]
Hi, I am new to the firenvim and this should be the killer feature I really want to have. I installed firenvim plug in for neovim and install firenvim addon for my firefox. Everything looks good in the beginning: when I click the textarea in the browser, the Firenvim overlay show up. However, once I finished typing, I press `<ESC>:wq`, and it shows the error message of 'No file name'.

I also realize the Firenvim can't read my init.vim setting. Meaning that my setting in init.vim does not take effect. Then I source init.vim in Firenvim overlay, it takes effect then, but ':wq' still can't bring the typed text back to the text area.

I checked the trouble shooting. But can't find the solution for me since my issue is not about no Fireevim overlay but can't bring the text back to Text Area... 

Please help!
 
- OS Version: macOS Catalina 10.15.3
- Browser Version: Firefox Browser 72.0.2
- Browser Addon Version: 0.1.20
- Neovim Plugin Version: I don't know how to check the the Plugin version but I just installed the latest one today.

**Comment by glacambre** [comment]
Hi :)

>However, once I finished typing, I press <ESC>:wq, and it shows the error message of 'No file name'.

This sounds like there is a problem with your init.vim. What happens when you press `<C-g>` in normal mode? What about `:w /tmp/x` and then `:q`? What does `:ls` print?

>I also realize the Firenvim can't read my init.vim setting.

If this was true, you wouldn't be able to use firenvim, as the neovim plugin wouldn't be loaded :).

>I checked the trouble shooting. But can't find the solution for me since my issue is not about no Fireevim overlay but can't bring the text back to Text Area...

Could you try [this](https://github.com/glacambre/firenvim/blob/master/TROUBLESHOOTING.md#make-sure-its-not-your-initvim) anyway?

**Comment by kuochang** [comment]
:ls prints `1 %a [no name]`

Thanks for response. After hourly checking, I realized it's my bad! I didn't carefully read the installation guide and I didn't do the instruction of "from resource", i.e.('npm install' and all that...). I thought it's the option at my first glance.:) 

Also, for somehow my colorscheme jellybean might conflict with the firenvim Overlay. I switch to other color scheme and it starts to behave normally.
BTW, my neovim is still "hybrid" since I put some settings in the old .vimrc and some in init.vim. Might take time to organize all setting into init.vim.

In short, the issue was resolved after npm install and change the 'jellybean' color scheme to 'slate', although I am still featuring out how can 'airline' and 'Chinese character' behave properly...:) Cheers!
